src/constants/index.js

```diff
@@ -0,0 +1,9 @@
+import icons from './icons';
+import { COLORS, SIZES, FONTS } from './theme';
+
+const theme = { COLORS, SIZES, FONTS };
+
+export {
+    theme,
+    icons
```

Comment:
The same about indentation and ESLint, comments about that that I left in the MainStackNavigator.js file.